packages/prisma/index.ts

```diff
@@ -25,7 +26,8 @@ export const customPrisma = (options?: Prisma.PrismaClientOptions) =>
     .$extends(usageTrackingExtention())
     .$extends(excludePendingPaymentsExtension())
     .$extends(bookingIdempotencyKeyExtension())
-    .$extends(withAccelerate());
+    .$extends(withAccelerate())
+    .$extends(withOptimize());
```

Comment:
You will need to pass in the API key here  `withOptimize({ token: process.env.OPTIMIZE_API_KEY })`

Reply:
Last time I checked the withOptimize didn't require an API key. Can you send me the documentation that you are referring to?

Reply:
Hi @harshsbhat, we launched a new version of Optimize recently. You can read more in the [docs](https://www.prisma.io/docs/optimize/getting-started)!